solutions/standard/variables.tf

```diff
@@ -80,6 +80,24 @@ variable "public_engine_enabled" {
   default     = false
 }
 
+variable "default_secret_group_name" {
+  type        = string
+  description = "Name to give the secrets group automatically created when provisioning a new Secrets Manager instance."
+  default     = "default-group"
+}
+
+variable "default_access_group_name" {
+  type        = string
+  description = "Name to give the access group automatically created when provisioning a new Secrets Manager instance."
+  default     = "secrets_manager_group"
```

Comment:
The name should probably have secrets reader in it since that is the role you are granting to this access group